packages/frontend/src/components/accounts/recovery_setup/SetupRecoveryMethod.js

```diff
@@ -195,9 +199,28 @@ class SetupRecoveryMethod extends Component {
             await validateSecurityCode(accountId, method, securityCode);
             await saveAccount(accountId, recoveryKeyPair);
 
+            // IDENTITY VERIFIED FUNDED ACCOUNT
+            if (DISABLE_CREATE_ACCOUNT && !fundingOptions && ENABLE_IDENTITY_VERIFIED_ACCOUNT) {
+                try {
+                    // Call function directly to handle error silently
```

Comment:
`createIdentityFundedAccount` action has an implicit `showAlert()` on it, so it is impossible for this code to  handle the error silently -- it'll redirect the user to the implicit fund flow, but they will see a global alert pop too. For context, `createNewAccount` action used to have an implicit `showAlert` attached to it also, but I had to remove it to make the logic below this to handle `NotEnoughBalance` actually be a silent redirect without an error (it had the same problem)